vars_geoportailv3.yaml

```diff
@@ -22,6 +22,9 @@ vars:
     # closure paths
     closure_library_path: 'process.stdout.write(require("openlayers/node_modules/closure-util").getLibraryPath())'
 
+    servers:
+        my_maps: http://map.geoportail.lu/my_maps
```

Comment:
this should be http://ws.geoportail.lu/mymaps (cf https://github.com/Geoportail-Luxembourg/geoportailv3/issues/131)